### PR TITLE
Circleci 2 machine

### PR DESCRIPTION
**Pull request by kchawla-pi**
Our CircleCI v2.0 config.yml using Docker didn't workout. 
@KamalakerDadi created a new circleci config.yml using machine. 
We keep getting `wget command not found` error when running the `continuous_integration/install.sh`
Adding `apt install --force wget` to config.yml didn't help.
Now trying to install wget from with the shell script `continuous_integration/install.sh`.

### CODE REVIEW

**Comment by KamalakerDadi** [comment]
Hi @lesteve Can we have some of your time to get this CircleCI updated to version 2.0 ? We are blocked by the ```wget command not found```.

Thanking you a lot.

**Comment by lesteve** [comment]
I'll try to take a look in a few days. Ping me if I forget.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      # Installing required packages for `make -C doc check command` to work.
+      - run: sudo -E apt-get -yq update
+      - run: sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install dvipng texlive-latex-base texlive-latex-extra wget
+      - save_cache:
+          key: v1-datasets-{{ .Branch }}
+          paths:
+            - ~/nilearn_data
+
+      # Moving to nilearn directory before performing the installation.
+      - run: cd nilearn
```
Comment (KamalakerDadi) [inline .circleci/config.yml]: This is not needed IMO.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#      - run: wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh
+#      - run: chmod +x ~/miniconda.sh && ~/miniconda.sh -b
+#      - run: export PATH=$HOME/miniconda2/bin:$PATH
+#      - run: echo $PATH
+#      - run: conda update --quiet --yes conda
+      - run:
+          command: ./continuous_integration/install.sh
+          environment:
```
Comment (KamalakerDadi) [inline .circleci/config.yml]: Try to setup this environment right after machine or docker image. It should not give ```wget command not found``` error.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+version: 2
+jobs:
+  build:
+    machine:
```
Comment (KamalakerDadi) [inline .circleci/config.yml]: I think we can switch to docker: with image: circleci/python: 3.5 or 3.6

File: .circleci/config.yml (hunk 2 above)
Comment (KamalakerDadi) [reply .circleci/config.yml]: I tried to build this locally. Atleast I am not getting the weird wget command not found error after building an environment before.

**Comment by KamalakerDadi** [comment]
>I'll try to take a look in a few days. Ping me if I forget.

Thanks @lesteve

**Pull request closed without merging**